web/src/main/java/by/ilyin/web/config/SecurityConfig.java

```diff
@@ -1,23 +1,38 @@
 package by.ilyin.web.config;
 
+import lombok.RequiredArgsConstructor;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpMethod;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.http.SessionCreationPolicy;
 import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
 
 @Configuration
 @EnableWebSecurity
+@RequiredArgsConstructor
 public class SecurityConfig {
+    private final JWTFilter jwtFilter;
 
     @Bean
     public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Exception {
         http
                 .csrf().disable()
                 .authorizeRequests()
-                .antMatchers("/api/about", "/api/users", "/api/users/*").permitAll();
-
+                .antMatchers(HttpMethod.POST, "/api/users").hasRole("ADMIN")
+                .antMatchers(HttpMethod.DELETE, "/api/users").hasRole("ADMIN")
+                .antMatchers(HttpMethod.GET, "/api/users").hasAnyRole("ADMIN", "DISPATCHER")
+                .antMatchers(HttpMethod.POST, "/api/users/{id}").hasRole("ADMIN")
+                .antMatchers(HttpMethod.GET, "/api/users/{id}").hasRole("ADMIN")
+                .antMatchers("api/logout").authenticated()
```

Comment:
@YauhenIlyin where did you find it in the ticket???

Reply:
Unless not only the authenticated user can be logged out?